src/types/emission.raw.ts

```diff
@@ -16,7 +16,9 @@ export const EmissionData = z.object({
   voteEmission: z.number(),
   percentUsdValue: z.number(),
   totalBribes: z.number(),
+  totalExternalBribes: z.number(),
   avgBribeRoiInPercent: z.number(),
+  avgExternalBribeRoiInPercent: z.number(),
```

Comment:
please make it '.optional()' to avoid conflicts with old data, that doesn't have this data